Core/Systems/MetaballSystem/MetaballActor.cs

```diff
@@ -14,6 +14,8 @@ public abstract class MetaballActor : IOrderedLoadable
 		/// </summary>
 		public virtual Color OutlineColor => Color.Black;
 
+		public virtual bool OverEnemies => false;
```

Comment:
A documentation comment on what this does would be nice

Reply:
Fixed